heron/config/src/yaml/conf/ecs/statemgr.yaml

```diff
@@ -0,0 +1,41 @@
+# local state manager class for managing state in a persistent fashion
+heron.class.state.manager:        com.twitter.heron.statemgr.zookeeper.curator.CuratorStateManager
+
+#Un-comment the line below and add ZK IP address
+#heron.statemgr.connection.string: "xx.xx.xx.xx:2181"
+
+
+# path of the root address to store the state in a local file system
```

Comment:
It does expect zookeeper to be running on one of the ec2 instances. As suggested I will put up a document explaining the offline set ups on Amazon ec2 and the following steps on the scheduler.